edx_proctoring/backends/backend.py

```diff
@@ -82,3 +82,10 @@ def get_attempt(self, attempt):
             dict: backend exam attempt object
         """
         return attempt
+
+    # pylint: disable=unused-argument
+    def get_instructor_url(self, course_id, user, exam_id=None, attempt_id=None):
```

Comment:
Why isn't this an abstract method? Because this is the default behavior?

Reply:
Yeah, it'll be the default behavior to do nothing. Making it an abstract method is a philosophical decision. Personally, I think having this base class be abstract doesn't help very much.